config/filter.d/dovecot.conf

```diff
@@ -14,6 +14,7 @@ failregex = ^%(__prefix_line)s(?:%(__pam_auth)s(?:\(dovecot:auth\))?:)?\s+authen
             ^%(__prefix_line)s(?:Info|dovecot: auth\(default\)|auth-worker\(\d+\)): pam\(\S+,<HOST>\): pam_authenticate\(\) failed: (User not known to the underlying authentication module: \d+ Time\(s\)|Authentication failure \(password mismatch\?\))\s*$
             ^%(__prefix_line)s(?:auth|auth-worker\(\d+\)): (?:pam|passwd-file)\(\S+,<HOST>\): unknown user\s*$
             ^%(__prefix_line)s(?:auth|auth-worker\(\d+\)): Info: ldap\(\S*,<HOST>,\S*\): invalid credentials\s*$
+            ^%(__prefix_line)sdovecot\[\d+]: (imap|pop3-login): Disconnected \(no auth attempts in \d secs\): user=<>, rip=<HOST>, lip=(\d{1,3}\.){3}\d{1,3}. TLS handshaking: (Disconnected|SSL_accept\(\) failed: Unknown error), session=<\S+>?\s*$
```

Comment:
It is a prelude to DDOS.  As with the ssh connects without an attempt to login below, there is no valid reason for a client to repeatedly connect to the pop3/imap server without an attempt to login.